### PR TITLE
linux/t*: add information link

### PR DESCRIPTION
**Pull request by CleanMachine1**
for #6062

### CODE REVIEW

File: pages/linux/trap.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 > Automatically execute commands after receiving signals by processes or the operating system.
 > Can be used to perform cleanups for interruptions by the user or other actions.
+> More information: N/A
```
Comment (CleanMachine1) [inline pages/linux/trap.md:5]: Need some help with this one

File: pages/linux/trap.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 > Automatically execute commands after receiving signals by processes or the operating system.
 > Can be used to perform cleanups for interruptions by the user or other actions.
+> More information: N/A.
```
Comment (bl-ue) [inline pages/linux/trap.md:5]: Why not https://manned.org/trap?
```suggestion
> More information: <https://manned.org/trap>.
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

File: pages/linux/trap.md (hunk 2 above)
Comment (CleanMachine1) [reply pages/linux/trap.md:5]: I cant tell if it is correct or not

Comment (bl-ue) [reply pages/linux/trap.md:5]: Looks like it to me.

**Review by sbrl — APPROVED** [review]
Thanks, @CleanMachine1!

**Review by SethFalco — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]

**Pull request merged**